apps/mobile/src/store/index.ts

```diff
@@ -5,23 +5,29 @@ import txHistory from './txHistorySlice'
 import activeSafe from './activeSafeSlice'
 import activeSigner from './activeSignerSlice'
 import signers from './signersSlice'
-import delegated from './delegatedSlice'
+import delegates from './delegatesSlice'
 import myAccounts from './myAccountsSlice'
 import notifications from './notificationsSlice'
 import addressBook from './addressBookSlice'
 import settings from './settingsSlice'
 import safes from './safesSlice'
+import safeSubscriptions from './safeSubscriptionsSlice'
 import biometrics from './biometricsSlice'
 import { cgwClient, setBaseUrl } from '@safe-global/store/gateway/cgwClient'
 import devToolsEnhancer from 'redux-devtools-expo-dev-plugin'
 import { GATEWAY_URL, isTestingEnv } from '../config/constants'
 import { web3API } from './signersBalance'
 import { setBaseUrl as setSDKBaseURL } from '@safe-global/safe-gateway-typescript-sdk'
 import { createFilter } from '@safe-global/store/utils/persistTransformFilter'
+import { setupMobileCookieHandling } from './utils/cookieHandling'
+import notificationsMiddleware from './middleware/notifications'
 
 setSDKBaseURL(GATEWAY_URL)
 setBaseUrl(GATEWAY_URL)
 
+// Set up mobile-specific cookie handling
+setupMobileCookieHandling()
+
```

Comment:
I couldn't get react-native-cookies to work, so I opted for manually dealing with the request cookie in the store package. (https://github.com/expo/expo/issues/6756)

Without sending a the cookie in the request header we end up not being authenticated on the server.